qemu/tests/multi_disk_wild_hotplug.py

```diff
@@ -90,35 +92,48 @@ def _hotunplug_images():
     params['start_vm'] = 'yes'
     env_process.preprocess_vm(test, params, env, params["main_vm"])
     vm = env.get_vm(params['main_vm'])
-    session = vm.wait_for_login(timeout=int(params.get("login_timeout", 360)))
-
-    error_context.context("Get images params", test.log.info)
-    _get_images_params()
-
-    disks_num = int(session.cmd("lsblk -d -n|wc -l", timeout=60))
-    test.log.info("There are total %d disks", disks_num)
-
-    guest_operation = params.get("guest_operation")
-    if guest_operation:
-        test.log.info("Run %s in guest ", guest_operation)
-        locals_var = locals()
-        locals_var[guest_operation]()
-
-    for n in range(repeat_num):
-        error_context.context("Start unplug loop:%d" % n, test.log.info)
-        _hotunplug_images()
-        time.sleep(unplug_time)
-        error_context.context("Start plug loop:%d" % n, test.log.info)
-        _hotplug_images()
-        time.sleep(plug_time)
-
-    error_context.context("Waiting for %d seconds" % wait_time, test.log.info)
-    time.sleep(wait_time)
-    error_context.context("Check disks in guest.", test.log.info)
-    # re-login in case previous session is expired
-    session = vm.wait_for_login(timeout=int(params.get("relogin_timeout", 60)))
-    new_disks_num = int(session.cmd("lsblk -d -n|wc -l", timeout=300))
-    test.log.info("There are total %d disks after hotplug", new_disks_num)
-    if new_disks_num != disks_num:
-        test.log.warning("Find unmatched disk numbers %d %d", disks_num,
-                         new_disks_num)
+    try:
+        session = vm.wait_for_login(timeout=int(params.get("login_timeout", 360)))
```

Comment:
the following method also work well. All look good to me.
timeout = params.get_numeric("login_timeout", 360)